modules/shared/src/main/scala/org/tessellation/statechannel/StateChannelSnapshot.scala

```diff
@@ -0,0 +1,7 @@
+package org.tessellation.statechannel
+
+import org.tessellation.security.hash.Hash
+
+trait StateChannelSnapshot {
```

Comment:
Seems that we don't use it anywhere anymore. Do we want to keep it in schema for the future?

Reply:
@TheMMaciek you're right. I've just removed it :+1: